### ISSUE TITLE
Enable aspects for remaining resources.

### ISSUE THREAD

**Issue by ayush987goyal** [cdklabs/cdk-monitoring-constructs#41]
Enable scope monitoring with aspects for the following remaining resources

- [ ] Cloudwatch Logs
- [ ] ECS Patterns
- [ ] ALB/NLB
- [ ] SQS with DLQ
- [ ] Stepfunction Integrations/Activity

**Comment by LarsFronius** [comment]
Would be awesome to see! Is there any inherent complexity one should think about when implementing aspects for these particular resources? I assume there was a reason it's not *yet* implemented.

**Comment by echeung-amzn** [comment]
I don't believe there was a particular reason that they haven't yet been implemented, but @ayush987goyal could confirm.

**Comment by ayush987goyal** [comment]
So these have not been implemented because of how the underlying facade implementation of the monitoring is done. For example 
* The ECS Patterns module is really tricky since it is difficult to identify all the resources and pass to the facade. 
* SQS with DLQ is also difficult since it is not entirely possible to related two queues to one another.
* Adding cloudwatch logs monitoring aspects based can add a lot of noise potentially.

**Comment by LarsFronius** [comment]
Thanks for the insight @ayush987goyal - that roughly matches what I had observed from glancing over it.

* ECS: I was thinking that maybe for the aspect side of the monitoring construct we could not work on the "patterns" but on the primitive of an ECS Service. I think we'd only only ever have CPU+Memory metrics but it's better than nothing?
* For SQS that approach would of course mean no DLQ connection as well - I'll have a look if there is any way to obtain the reference though. Currently it looks like they show up as individual queues on the dashboard.
* For cloudwatch logs I've gone down the route of adding **one** dashboard segment that automatically encompasses all log groups, so that they are not scattered as individual log groups on the dashboard. It looks somewhat clean for my case, but that may vary a lot on the individual stacks and purposes (in my case it's an ECS (Fargate) + ALB stack). I reckon that could be a configurable option on the monitoring aspect itself?

**Comment by RichiCoder1** [comment]
I mentioned this in #120, but I'm personally very much in favor of breaking ECS down into it's component parts for monitoring and not trying to work with the L2/L3 patterns.
